src/SFML/Window/DRM/InputImplUDev.hpp

```diff
@@ -0,0 +1,175 @@
+////////////////////////////////////////////////////////////
+//
+// SFML - Simple and Fast Multimedia Library
+// Copyright (C) 2016 Andrew Mickelson
+//
+// This software is provided 'as-is', without any express or implied warranty.
+// In no event will the authors be held liable for any damages arising from the use of this software.
+//
+// Permission is granted to anyone to use this software for any purpose,
+// including commercial applications, and to alter it and redistribute it freely,
+// subject to the following restrictions:
+//
+// 1. The origin of this software must not be misrepresented;
+//    you must not claim that you wrote the original software.
+//    If you use this software in a product, an acknowledgment
+//    in the product documentation would be appreciated but is not required.
+//
+// 2. Altered source versions must be plainly marked as such,
+//    and must not be misrepresented as being the original software.
+//
+// 3. This notice may not be removed or altered from any source distribution.
+//
+////////////////////////////////////////////////////////////
+
+#ifndef SFML_INPUTIMPLUDEV_HPP
+#define SFML_INPUTIMPLUDEV_HPP
```

Comment:
Might not know enough about Linux components, but is this still using `UDev`, i.e. does the name still make sense?

Reply:
@eXpl0it3r We're accessing `/dev/input/event` so yes, It's still handled by UDev daemon.